### DISCUSSION TITLE
advanced under Views [May 05]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2032] [Discord Threads]
> 🙋‍♂️ **ovizii**  
#### via Docker and I am wondering why I don't see the "advanced" section which is visible in every demo/screenshot on the right side below the "views" section.  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/969903389448224778/unknown.png)

**Comment by o1lab** [comment]
> 👨‍🚀 **wingkwong**  
#### the demo & screenshots r outdated. will update gradually  
#  
> 🙋‍♂️ **ovizii**  
#### ah, thanks, I was going nuts figuring out where this section had disappeared on my instance 🙂    
#### meanwhile I have found some of those "missing" features in some other places of nocodb  
![image0](https://cdn.discordapp.com/attachments/969903390186418176/971177068207431680/unknown.png)  
#  
> 👨‍🚀 **wingkwong**  
#### whare are those "missing" features?  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/969903390186418176)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
